cmd/plume/upload.go

```diff
@@ -86,8 +86,13 @@ func runUpload(args []string) int {
 		fmt.Fprintf(os.Stderr, "URL missing bucket name %v\n", bucket)
 		return 1
 	}
+
 	bucket = gsURL.Host
+	// add prefix
 	imageName = strings.TrimPrefix(gsURL.Path+"/"+imageName, "/")
+	// create equivalent image names for GS and GCE
+	imageNameGCE := gceSanitize(imageName)
```

Comment:
Although this scheme including the full gsURL is simple for now something like the following would be cleaner: `$USER-$BOARD-$imageName`